.travis.yml

```diff
@@ -52,6 +53,7 @@ jobs:
         - set -o errexit; source .travis/lint_06_script.sh
 # ARM
```

Comment:
I guess these comments could then be removed

Reply:
I just pushed a commit that removes these comments and moves the needed comments to beside the job name.